pyperformance/venv.py

```diff
@@ -24,6 +24,8 @@ def from_benchmarks(cls, benchmarks):
         for bench in benchmarks or ():
             filename = bench.requirements_lockfile
             self._add_from_file(filename)
+            if bench.setup_py:
+                self._add(os.path.dirname(bench.metafile))
```

Comment:
I'm not sure I get this one.  `setup_py` is not a boolean to indicate there is one, and then you pass the directory it contains to `pip` (not the file itself).  Maybe renaming this flag to `needs_build` would be more accurate (since that how `pip` see it -- this directory has something that needs to be built, and `setup.py` is just an implementation detail...)

Reply:
Sorry for the confusion.  My point is that we should record the directory name on the Benchmark object rather than using `os.path.dirname()` here.  We could call it something like `setup_py_dir` if you prefer.

Reply:
I've revisited this in https://github.com/python/pyperformance/pull/197#discussion_r930154904.